reference/conanfile/attributes.rst

```diff
@@ -73,6 +73,27 @@ The profile entries have priority.
           self.output.info("Custom var1: %s" % self.conf.get("user.custom.var1"))
 
 
+Output
+------
+
+.. _conanfile_output:
+
+Output contents
+---------------
+
+Use the `self.output` to print contents to the output.
+
+..  code-block:: python
+
+   self.output.success("This is good, should be green")
+   self.output.info("This is neutral, should be white")
+   self.output.warning("This is a warning, should be yellow")
+   self.output.error("Error, should be red")
+   self.output.rewrite_line("for progress bars, issues a cr")
```

Comment:
Lets drop the ``rewrite_line`` atm, I think we want to internally get rid of it.